src/assets.ts

```diff
@@ -244,28 +179,91 @@ export class AssetGenerator {
     private createBuildTaskDescription(): tasks.TaskDescription {
         let buildPath = '';
         if (this.hasProject) {
-            buildPath = path.join('${workspaceRoot}', path.relative(this.rootPath, this.projectFilePath));
+            buildPath = path.join('${workspaceFolder}', path.relative(this.rootPath, this.projectFilePath));
         }
 
         return {
             taskName: 'build',
-            args: [util.convertNativePathToPosix(buildPath)],
-            isBuildCommand: true,
+            command: 'dotnet',
+            type: 'process',
+            args: ['build', util.convertNativePathToPosix(buildPath)],
             problemMatcher: '$msCompile'
         };
     }
 
     public createTasksConfiguration(): tasks.TaskConfiguration {
         return {
-            version: '0.1.0',
-            command: 'dotnet',
-            isShellCommand: true,
-            args: [],
+            version: "2.0.0",
             tasks: [this.createBuildTaskDescription()]
         };
     }
 }
 
+export function createWebLaunchConfiguration(programPath: string, workingDirectory: string): string {
+    return `
+{
+"name": ".NET Core Launch (web)",
```

Comment:
Any reason not to indent this like you did below to make it a bit more readable?

Reply:
Oops. Lost indentation when copying.